test/fixtures/Test-requests_3022673737/Getting-XDM-README-with-GitHub-token_2914372360/recording.har

```diff
@@ -0,0 +1,161 @@
+{
+  "log": {
+    "_recordingName": "Test requests/Getting XDM README",
+    "creator": {
+      "comment": "persister:fs",
+      "name": "Polly.JS",
+      "version": "1.4.1"
+    },
+    "entries": [
+      {
+        "_id": "8801a2001578e12b49266529916516fa",
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 0,
+          "cookies": [],
+          "headers": [
+            {
+              "name": "authorization",
+              "value": "token mytesttoken"
+            },
+            {
+              "name": "host",
+              "value": "raw.githubusercontent.com"
+            }
+          ],
+          "headersSize": 110,
+          "httpVersion": "HTTP/1.1",
+          "method": "GET",
+          "queryString": [],
+          "url": "https://raw.githubusercontent.com/adobe/xdm/master/README.md"
```

Comment:
It would be better to test this with an actual private repo ideally one where the content isn't *that* private.

Reply:
done